configs/swagger.go

```diff
@@ -0,0 +1,14 @@
+package configs
+
+import (
+	docs "yab-explorer/docs"
+)
+
+func InitSwagger(apiPort string) {
+	docs.SwaggerInfo.Title = "YAB Explorer API"
+	docs.SwaggerInfo.Description = "YAB Explorer API"
+	docs.SwaggerInfo.Version = "1.0"
+	docs.SwaggerInfo.Host = "localhost:" + apiPort
```

Comment:
Is there a way to add multiple hosts?
Ie here https://api.yetanotherswap.com/docs/ we have 3 servers available